tests/test_error_handling.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from lsst.cm.tools.core.handler import Handler
+from lsst.cm.tools.db.sqlalch_interface import SQLAlchemyInterface
+
+
+def test_error_handling() -> None:
+    try:
+        os.unlink("test_error_handling.db")
+    except OSError:  # pragma: no cover
+        pass
+    os.system("\\rm -rf archive_test")
```

Comment:
I think `shutil.rmtree()` library function would be preferred for cases like this (doesn't need to spawn a an entire sub-process).